### ISSUE TITLE
Save Files (Google Drive)

### ISSUE THREAD

**Issue by Fleker** [ITVlab/TvFilePicker#5]
Similar to #4, there should be an easy method to write a file to Google Drive with a particular folder path instead of locally.

**Comment by JuanRodriguez32** [comment]
I can help you with this. Let me know if you are interested.

**Comment by Fleker** [comment]
Yeah that would be great. Part of this project deals with being able to
navigate the Google drive file structure and adding some UI to handle
saving. This saving should be modular enough to work with the local file
system or Google drive.
